redisinsight/api/src/modules/database/database.service.ts

```diff
@@ -148,32 +180,35 @@ export class DatabaseService {
     let database = merge({}, oldDatabase, dto);
 
     try {
-      database = await this.databaseFactory.createDatabaseModel(database);
+      if (Object.keys(omitBy(dto, isUndefined)).some((field) => this.connectionFields.includes(field))) {
```

Comment:
I like this check. Well done
Anyway let's move it to own method (even static) since I already found 2 places with this copy paste

Reply:
Agreed. Thank you